Docs/reference/content/what_is_new.md

```diff
@@ -8,6 +8,15 @@ title = "What's New"
   pre = "<i class='fa fa-star'></i>"
 +++
 
+## What's New in 2.8.0
+
+Some of the changes in 2.8.0 include:
+
+* A number of minor bug fixes
+* Update dependency on System.Net.Security to 4.3.2
+* Update dependency on System.Runtime.InteropServices.RuntimeInformation to 4.3.0
+* Update dependency on DnsClient to 1.2.0 and work around breaking changes in the latest version of DnsClient
+
```

Comment:
Nothing else really jumped out at me looking at the JIRA tickets fixed in 2.8.0.

Reply:
Should the three dependency updates should be mentioned on the upgrading page?  The rationale is that the older versions caused compatibility problems for some users, so the newer versions could for other people